### DISCUSSION TITLE
No event rules tab

### DISCUSSION THREAD

**Discussion by blobf75** [netbox-community/netbox#15747] [General]
Hi,
I've been using Netbox 3.7.4 for a while now, and for some reason, the "Event rules" menu is missing from me.
What could be the reason ?

![image](https://github.com/netbox-community/netbox/assets/71228481/2cb2c104-7ad3-4ac0-aa1f-6ca7d37bb82b)

**Comment by candlerb** [comment]
Are you logged in as superuser? If not, do you have permission granted on Extras > Event Rule?

**Reply by blobf75** [reply]
I have an active Directory account with administrator privileges but probably not superuser. I do not see the Extras > Event rules tab.

**Reply by candlerb** [reply]
Then look at your mapping between AD groups and Netbox groups, and what permissions are granted to those Netbox groups.
